### DISCUSSION TITLE
GitHub Actions fail

### DISCUSSION THREAD

**Discussion by obaume** [web-classroom/web-classroom.github.io#15] [Q&A : Pratique / Labos]
Les tests ne passent dans le github actions à cause des permissions mocha:
```
> 07-websocket@1.0.0 test
  > mocha --experimental-modules
  
  sh: 1: mocha: Permission denied
```

**Comment by OlivierLmr** [comment]
Est-ce que vous avez modifié le fichier `package.json` ? Et est-ce que vous connaissez d'autres personnes à qui ça arrive ?

**Comment by OlivierLmr** [comment]
Aussi, localement, est-ce que les tests passent ?
